packages/web/src/scss/helpers/_accessibility.scss

```
@@ -3,3 +3,19 @@
 .accessibility-hidden {
     @include accessibility.hide-text();
 }
+
+[aria-expanded='true'] .accessibility-expanded {
+    display: initial;
+}
+
+[aria-expanded='false'] .accessibility-expanded {
+    display: none;
+}
+
+[aria-expanded='true'] .accessibility-collapsed {
+    display: none;
+}
+
+[aria-expanded='false'] .accessibility-collapsed {
+    display: initial;
+}
```

For the sake of efficiency, these can be combined into groups:

```suggestion
[aria-expanded='true'] .accessibility-expanded,
[aria-expanded='false'] .accessibility-collapsed {
    display: initial;
}

[aria-expanded='false'] .accessibility-expanded,
[aria-expanded='true'] .accessibility-collapsed {
    display: none;
}
```

---

fixed